wifiphisher/common/accesspoint.py

```diff
@@ -151,3 +169,30 @@ def on_exit(self):
         # sleep 2 seconds to wait all the hostapd process is
         # killed
         time.sleep(2)
+
+    def read_connected_victims_file(self):
+        """
```

Comment:
Docstring needs to comply with [pep257](https://www.python.org/dev/peps/pep-0257/).